Wordle.Api/Wordle.Api/Services/WordOfTheDayService.cs

```diff
@@ -60,6 +61,25 @@ public async Task<string> GetWordOfTheDay(DateOnly date)
         return wordOfTheDay.Word!.Text;
     }
 
+    //Attempted Api Service for getting stats
+    /* 
+    public async Task<GameStatsDto> GetWordStats(string word)
+    {
+      var wordStats = await Db.Words
+        .Where(w => w.Text.Equals(word, StringComparison.OrdinalIgnoreCase))
```

Comment:
the where implementation might not be best here. we used WordOfTheDay? word = await Db.WordsOfTheDays
            .Include(wotd => wotd.Games)
            .FirstOrDefaultAsync(wotd => wotd.Date == dateOnly);

Reply:
Refactored, thank you!